core/src/test/java/greencity/controller/SuperAdminControllerTest.java

```diff
@@ -361,8 +362,15 @@ void deactivateTariffTest() {
     }
 
     @Test
-    void setLimitDescription() throws Exception {
-        mockMvc.perform(patch(ubsLink + "/setLimitDescription/{courierId}", 1L))
+    @SneakyThrows
+    void setLimitDescriptionTest() throws Exception {
+        SetLimitDesriptionDto dto = ModelUtils.getLimitDescriptionDto();
+        ObjectMapper objectMapper = new ObjectMapper();
+        String responseJSON = objectMapper.writeValueAsString(dto);
+        mockMvc.perform(patch(ubsLink + "/setLimitDescription/{tariffId}", 1L)
+            .principal(principal)
+            .content(responseJSON)
```

Comment:
I do not see any annotation that endpoint ` /setLimitDescription/{tariffId}`  gets body
Should we modify the endpoint too?